src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/TseAdminService.java

```diff
@@ -0,0 +1,329 @@
+                        new TSEAdminEmailRecipientsData(
+                            emailToRespondentTemplateId,
+                            respondentSumTypeItem.getValue().getRespondentEmail());
+                    respondentDetails.setRecipientName(respondentSumTypeItem.getValue().getRespondentName());
+
+                    emailsToSend.add(respondentDetails);
+                }
+            }
+        }
+
+        // if claimant only or both parties: send Claimant Decision Email
+        if (CLAIMANT_ONLY.equals(caseData.getTseAdminSelectPartyNotify())
+                || BOTH.equals(caseData.getTseAdminSelectPartyNotify())) {
+            String claimantEmail = caseData.getClaimantType().getClaimantEmailAddress();
+            String claimantName = caseData.getClaimantIndType().getClaimantFirstNames()
+                + " " + caseData.getClaimantIndType().getClaimantLastName();
+
+            if (claimantEmail != null) {
+                TSEAdminEmailRecipientsData claimantDetails =
+                    new TSEAdminEmailRecipientsData(emailToClaimantTemplateId, claimantEmail);
+                claimantDetails.setRecipientName(claimantName);
+
+                emailsToSend.add(claimantDetails);
+            }
+        }
+
+        for (final TSEAdminEmailRecipientsData emailRecipient : emailsToSend) {
+            emailService.sendEmail(
+                emailRecipient.getRecipientTemplate(),
+                emailRecipient.getRecipientEmail(),
+                buildPersonalisation(caseNumber, caseId, emailRecipient.getRecipientName()));
+        }
+    }
+
+    private Map<String, String> buildPersonalisation(String caseNumber, String caseId, String recipientName) {
+        Map<String, String> personalisation = new ConcurrentHashMap<>();
+        personalisation.put("caseNumber", caseNumber);
+        personalisation.put("caseId", caseId);
+        personalisation.put("name", recipientName);
+        return personalisation;
+    }
+
+    /**
+     * Clear Tse Admin Record a Decision Interface data from caseData.
+     * @param caseData in which the case details are extracted from
+     */
+    public void clearTseAdminDataFromCaseData(CaseData caseData) {
+        caseData.setTseAdminSelectApplication(null);
+        caseData.setTseAdminTableMarkUp(null);
+        caseData.setTseAdminEnterNotificationTitle(null);
+        caseData.setTseAdminDecision(null);
+        caseData.setTseAdminDecisionDetails(null);
+        caseData.setTseAdminTypeOfDecision(null);
+        caseData.setTseAdminIsResponseRequired(null);
+        caseData.setTseAdminSelectPartyRespond(null);
+        caseData.setTseAdminAdditionalInformation(null);
+        caseData.setTseAdminResponseRequiredYesDoc(null);
+        caseData.setTseAdminResponseRequiredNoDoc(null);
+        caseData.setTseAdminDecisionMadeBy(null);
+        caseData.setTseAdminDecisionMadeByFullName(null);
+        caseData.setTseAdminSelectPartyNotify(null);
+    }
+
+    // TODO -
```

Comment:
Review this todo list I believe some of them should be available to do now

Reply:
Resolved by Robin

Reply:
can the todo be removed?